ext/mysqli/tests/mysqli_constants.phpt

```diff
@@ -181,10 +181,6 @@ if ($IS_MYSQLND) {
     $expected_constants["MYSQLI_TYPE_JSON"]	= true;
 }
 
-if (($version > 80021 && $constants['mysqli']['MYSQLI_IS_MARIADB']) || $IS_MYSQLND) {
```

Comment:
It seems to me that the `$IS_MYSQLND` is not necessary anymore, but I didn't want to clean it up as part of this PR. Feel free to do so, or I can also do it later if you are busy.

Reply:
Done #8919

Reply:
I mean, I removed the constant, but didn't want to remove all the `$IS_MYSQLND` references in this test (or are they still needed?).

Reply:
It's always mysqlnd now. We don't allow any other library for mysqli.